### PR TITLE
Add option to list ordered totals

### PR DESCRIPTION
**Pull request by martinmosko**

### CODE REVIEW

File: src/main/java/com/gooddata/md/report/Total.java [hunk 1]
```diff
@@ -41,4 +44,16 @@ public static Total of(String total) {
                     e);
         }
     }
+
+    public static List<Total> orderedValues() {
+        List<Total> orderedTotals = new ArrayList<>();
```
Comment (jimirocks) [inline src/main/java/com/gooddata/md/report/Total.java]: user rather `Arrays.asList`

Comment (jimirocks) [reply src/main/java/com/gooddata/md/report/Total.java]: and also add javadoc explaining this refers the order the totals are returned by executor

Comment (martinmosko) [reply src/main/java/com/gooddata/md/report/Total.java]: changed & added javadoc

File: src/test/groovy/com/gooddata/md/report/TotalTest.groovy [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+import static com.gooddata.md.report.Total.AVG
+import static com.gooddata.md.report.Total.MAX
+import static com.gooddata.md.report.Total.MED
+import static com.gooddata.md.report.Total.MIN
+import static com.gooddata.md.report.Total.NAT
+import static com.gooddata.md.report.Total.SUM
+
+class TotalTest extends Specification {
+    def "should for Total.of(...) throw exception on invalid name"() {
```
Comment (jimirocks) [inline src/test/groovy/com/gooddata/md/report/TotalTest.groovy]: please reword the test summary into the shape of sentence

Comment (martinmosko) [reply src/test/groovy/com/gooddata/md/report/TotalTest.groovy]: done

**Pull request merged**